easybuild/easyconfigs/f/fastStructure/fastStructure-1.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,53 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+name = "fastStructure"
+version = "1.0"
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://rajanil.github.io/fastStructure/"
+description = """
+fastStructure is a fast algorithm for inferring population structure
+from large SNP genotype data. It is based on a variational Bayesian
+framework for posterior inference and is written in Python2.x.
+"""
+docurls = ["https://github.com/rajanil/fastStructure"]
+
+toolchain = {"name": "foss", "version": "2019a"}
+
+# https://github.com/rajanil/fastStructure/archive/
+github_account = "rajanil"
+source_urls = [GITHUB_SOURCE]
+sources = ["v%(version)s.tar.gz"]
+checksums = ['f1bfb24bb5ecd108bc3a90145fad232012165c1e60608003f1c87d200f867b81']
+
+dependencies = [
+    ("Python", "2.7.15"),
+    ("Cython", '0.27.3'),
```

Comment:
Isn't `Cython` just a build dep?

Also, please add a comment to clarify why you're depending on a specific `Cython` version.